### ISSUE TITLE
Question: Does the scoop/scoop-extras repos update themselves on daily basis?

### ISSUE THREAD

**Issue by winteryoung** [lukesampson/scoop#1718]
Just curious. I see updating the manifests requires committing to the repos. So is this process triggered by human or by some daemon processes running in the background of someone's PC?

The marginal cost of maintaining a manifest is greatly reduced by autoupdate. I want to know if the last part of the process has been automated or not.

**Comment by r15ch13** [comment]
Yes it's a cronjob on my server. I created a docker service that runs auto-pr.ps1 every hour: https://github.com/r15ch13/Excavator

**Comment by winteryoung** [comment]
thx

**Comment by r15ch13** [comment]
Ha! Found a bug in my docker service ... extras and versions bucket didn't get updated for three days 😁

**Comment by rasa** [comment]
@r15ch13: if I gave you write access, would you wanna add https://github.com/rasa/scoops to your docker service?

**Comment by r15ch13** [comment]
I could. Another idea would be to create an scoop docker image where every bucket can be based on. So anyone can easily create a service for self-made buckets.
